### PR TITLE
Fix for unable to invoke the APIs after upgrading the subscription tier after throttling out

### PR DESCRIPTION
**Pull request by binodmx**
This PR resolves the issue https://github.com/wso2/product-apim/issues/11372

### CODE REVIEW

**Review by tharindu1st — APPROVED** [review]

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIManagerConfiguration.java [hunk 1]
```diff
@@ -1023,6 +1023,14 @@ private void setThrottleProperties(OMElement element) {
                 throttleProperties.setEnabledSubscriptionLevelSpikeArrest(JavaUtils.isTrueExplicitly
                         (enabledSubscriptionLevelSpikeArrestElement
                                 .getText()));
+            }
+            // Check Immediate SubscriptionPolicyUpdate
+            OMElement enableImmediateSubscriptionPolicyUpdateElement =
+                    throttleConfigurationElement.getFirstChildWithName(new QName(APIConstants.AdvancedThrottleConstants
+                            .ENABLE_IMMEDIATE_SUBSCRIPTION_POLICY_UPDATE));
+            if (enableImmediateSubscriptionPolicyUpdateElement != null) {
+                throttleProperties.setEnableImmediateSubscriptionUpdate(JavaUtils.isTrueExplicitly
```
Comment (sdkottegoda) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIManagerConfiguration.java:1032]: Can't we simply use Boolean.parseBoolean()?

Comment (binodmx) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIManagerConfiguration.java:1032]: Front porting changes from 3.2.0.

**Review by tharindu1st — APPROVED** [review]

**Pull request merged**